data/roles.js

```diff
@@ -629,157 +648,163 @@ const roleData = {
         "Diabolist": {
             alignment: "Mafia",
             description: [
-                "Can choose a victim and a target each night.",
-                "If the victim votes for the target, the victim will die."
+                "Chooses a player to be a victim and a target each night.",
+                "If the victim votes for the target in the village meeting the following day, the victim will die."
             ]
         },
         "Tailor": {
             alignment: "Mafia",
             description: [
-                "Gives out suits that make the wearer appear to be a different role."
+                "Gives out a suit each night that disguise the wearer's role identity.",
+                "Suits can be selected from any role within the current game."
             ],
         },
         "Actress": {
             alignment: "Mafia",
             description: [
-                "Learns a targets role and appears to be that role."
+                "Visits a player to appears as their role.",
+                "Learns chosen player's role."
             ],
         },
         "Prosecutor": {
             alignment: "Mafia",
             description: [
-                "Votes weight is 2."
+                "Vote weight is worth 2 players votes in village meeting."
```

Comment:
Same as Judge, the "players" seems unnecessary here. But not a necessary change, this is just my opinion.